.travis.yml

```diff
@@ -77,4 +77,4 @@ after_success:
 addons:
     firefox: "latest-beta"
     postgresql: "10"
-    mariadb: "10.4"
+    mariadb: "10.3"
```

Comment:
I'd rather drop testing for 16.

Reply:
Reverted mariadb to 10.4 and changed the stable16 test to use pgsql that way we will still see comparability issues of news with nc16